herald/__init__.py

```diff
@@ -1,7 +1,7 @@
 """
 Notification classes. Used for sending texts and emails
 """
-__version__ = '0.2'
+__version__ = '0.2.1'
```

Comment:
Since we haven't release 0.2 yet, we can just add this to the 0.2 release. Thanks!